eng/scripts/Create-APIView.ps1

```diff
@@ -138,6 +159,68 @@ function Invoke-SwaggerAPIViewParser {
     }
 }
 
+<#
+.DESCRIPTION
+  Invoke the TypeSpec parset to generate APIView tokens.
+
+.PARAMETER Type
+  New or Baseline TypeSpec APIView tokens.
+
+.PARAMETER ProjectPath
+  The TypeSpec Project path.
+
+.PARAMETER ResourceProvider
+  The ResourceProvider Name.
+
+.PARAMETER Tag
+  The Tag to use for generating the APIView Tokens.
+
+.PARAMETER TokenDirectory
+  The directory to store the generated APIView Tokens.
+
+.OUTPUTS
+  The resource provider name.
+#>
+function Invoke-TypeSpecAPIViewParser {
+  param (
+      [ValidateSet("New", "Baseline")]
+      [Parameter(Mandatory = $true)]
+      [string]$Type,
+      [Parameter(Mandatory = $true)]
+      [string]$ProjectPath,
+      [Parameter(Mandatory = $true)]
+      [string]$ResourceProvider,
+      [Parameter(Mandatory = $true)]
+      [string]$TokenDirectory
+  )
+  $tempWorkingDirectoryName = [guid]::NewGuid().ToString()
+  $tempWorkingDirectoryPath = [System.IO.Path]::Combine($TempDirectory, $tempWorkingDirectoryName)
+  New-Item -ItemType Directory -Path $tempWorkingDirectoryPath > $null
+
+  try {
+    Write-Host "Installing required dependencies to generate API review"
+    npm ci
```

Comment:
Could the install be done earlier in the pipeline?

Reply:
Yes, these steps should be moved into the pipeline yaml.  You can use this template:

https://github.com/Azure/azure-rest-api-specs/blob/main/eng/pipelines/templates/steps/npm-install.yml